Duplicati/Library/SQLiteHelper/SQLiteLoader.cs

```diff
@@ -271,6 +271,13 @@ private static void OpenSQLiteFile(System.Data.IDbConnection con, string path)
             con.ConnectionString = "Data Source=" + path;
             con.Open();
 
+            // Enable write-ahead logging and memory mapped IO
+            using (System.Data.IDbCommand command = con.CreateCommand())
+            {
+                command.CommandText = "PRAGMA journal_mode = WAL;PRAGMA mmap_size=268435456;";

```

Comment:
We could make this only run on first DB creation, but then it wouldn't work for existing DBs.